source/expressions.tex

```diff
@@ -780,7 +780,7 @@
 where $\cv{}^j_i$ and $P^j_i$ are the components of
 the qualification-decomposition of $\tcode{T}j$.
 A prvalue of type \tcode{T1}
-can be converted to type \tcode{T2}
+can be converted to another type \tcode{T2}
```

Comment:
Since we have excluded the identity conversion from qualification conversion, [expr.const.cast] p7 should also have to be changed to adhere this modification

[expr.const.cast] p7 says
> A conversion from a type T1 to a type T2 casts away constness if T1 and T2 are different, there is a qualification-decomposition ([conv.qual]) of T1 yielding n such that T2 has a qualification-decomposition of the form
>> - [...]
>
> and there is **no qualification conversion** that converts T1 to
>> - [...]  named T3
> 

Consider this example:
````cpp
int* ptr = 0;
static_cast<void*>(ptr); 
````
In this example, `T1` is "pointer to int" while `T2` is "pointer to void". When n=1, `T3` is `pointer to void`, since `T2` and `T3` are the same types, hence there is no qualification conversion that converts `T2` to `T3`([conv.qual] p3 excludes identify conversions).  This is not the intent of the definition of the cast-away constness. 

[expr.const.cast] p7 should be changed to
> A conversion from a type T1 to a type T2 casts away constness if T1 and T2 are different, there is a qualification-decomposition ([conv.qual]) of T1 yielding n such that T2 has a qualification-decomposition of the form
>> - [...]
>
> and there is **no qualification conversion** that converts T1 to
>> - [...]  
>> 
> **if they are different types**
>